README.md

```diff
@@ -70,7 +74,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends.each do |i|
+  puts "Belle is friends with #{i}"
```

Comment:
:+1: This works, but what is a more semantic block variable? We tend to avoid using `i` outside the context of index or iteratee loops